assembly/test/channels.ts

```diff
@@ -17,5 +16,5 @@ export function _start(): void {
     for (let i = 0; i < data.length; i++) {
         assert(data[i] == result[i]);
     }
-    Console.log("[Pass] Basic Send/Receive\r\n");
+    console.log("[Pass] Basic Send/Receive\r\n");
```

Comment:
does `console.log` provided by assemblyscript append `\r\n` already?

Reply:
Good question given this output:

```

[Pass] Thread with simple value

[Pass] Simple thread with channel pass

[Pass] Thread with flat reference

[Pass] Thread with array

[Pass] Thread with typedarray

[Pass] Thread with static staticarray segment

[Pass] Thread with allocated staticarray segment
```

I think so ;-)